e2e/tests/negative/checksum_enabled_large_volume_with_multiple_rebuilding.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Documentation    -- Manual test plan --
+...              - https://longhorn.github.io/longhorn-tests/manual/pre-release/stability/checksum-enabled-large-volume/
+...              - 1. Create a 50 Gi volume. write around 30 Gi data into it.
+...              - 2. Enable the setting Snapshot Data Integrity.
+...              - 3. Keep writing in the volume continuously using dd command like 
+...              -    while true; do dd if=/dev/urandom of=t1 bs=512 count=1000 conv=fsync status=progress && rm t1; done.
+...              - 4. Create a recurring job of backup for every 15 min.
+...              - 5. Delete a replica and wait for the replica rebuilding.
+...              - 6. Repeat the steps of deletion of the replica and verify Longhorn doesn’t take more time than the first iteration.
+...
+...                == Not implemented ==
+...              - 7. Compare the performance of replica rebuilding from previous Longhorn version
+...              -    without the setting Snapshot Data Integrity.
+...              - 8. Verify the Longhorn manager logs, no abnormal logs should be present.
+
+
+Test Tags    manual    negative    longhorn-10711    large-size
```

Comment:
I think the purpose of tagging Longhorn issue number `longhorn-10711` is to track the original bug/feature ticket that this test is meant to cover. But in https://github.com/longhorn/longhorn/issues/10711, I couldn't clearly identify the related bug or feature. I assume it might be https://github.com/longhorn/longhorn/issues/4210, https://github.com/longhorn/longhorn/issues/4845 or https://github.com/longhorn/longhorn/issues/3198. Could you help confirm and add links to the ticket?

Alternatively, we could just update the tag to `longhorn-4210` (or the correct one), so the test case directly points to the relevant bug or feature ticket.